src/reference/modules/usage/pages/variables.adoc

```diff
@@ -633,7 +631,7 @@ The resulting properties will be:
 
 Agents can define variables:
 
-* For the Linux agent, https://docs.cfengine.com/docs/3.15/reference-special-variables-sys.html[CFEngine system variable] are available.
+* For the classic (Unix) agent, https://docs.cfengine.com/docs/3.18/reference-special-variables-sys.html[CFEngine system variable] are available.
```

Comment:
I think we chose to use the term "Linux" agent when we chose to use "Windows" (for DSC one)